.github/workflows/ci-sdk-as-test.yml

```diff
@@ -6,6 +6,9 @@ on:
       - synchronize
       - reopened
       - ready_for_review
+    paths:
+      - /sdk/assemblyscript
+      - /runtime
```

Comment:
my thinking on this one was that we would want to run the SDK tests whenever the runtime was updated to ensure SDK changes weren't needed to maintain compatibility–your call whether that's useful

Reply:
The SDK tests don't use the runtime, so that wouldn't help.  But the scenario you're thinking about is covered already, by the runtime tests in `/runtime/languages/assemblyscript/tests` and the project in the corresponding `testdata` folder.